decentraland/unity-renderer
# feat: send friend requests

## What does this PR change?

Adds a new popup to send a friend request including a custom message.
It is missing the test coverage, will do in another pr.

## How to test the changes?

1. Get near any other peer
2. Open the passport and then click the add friend button
3. A popup should appear allowing you to follow the process of sending a request
4. Open the chat
5. Right-click on any user name
6. Click on add friend button
7. Repeat step 3

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
